python/adelphi/tests/integration/__init__.py

```diff
@@ -22,6 +22,20 @@
 
 class SchemaTestMixin:
 
+    # Resource directory management logic
+    def cqlReferenceSchema(self, version):
+        return "tests/integration/resources/cql-schemas/{}.cql".format(version)
+
+
+    def nbReferenceYaml(self, version):
+        return "tests/integration/resources/nb-schemas/{}.yaml".format(version)
+
+
+    def nbBaseSchema(self):
+        return "tests/integration/resources/nb-base-schema.cql"
```

Comment:
Some file cleanup.  Patterns were developing in the tests around referencing base + expected CQL/YAML but many of the tests were still manipulating raw paths.  Goal here is to consolidate that logic in the framework so that it only has to be implemented once.